src/style.css

```diff
@@ -0,0 +1,109 @@
+body {
+  margin: 0;
+}
+
+.container {
+  padding: 1px;
```

Comment:
why? 😭

Reply:
otherwise the test does not want to pass 
but I will try to fix it 😄